graph/src/components/subgraph/host.rs

```diff
@@ -108,6 +109,16 @@ impl HostMetrics {
                 vec![0.1, 0.5, 1.0, 10.0, 100.0],
             )
             .expect("failed to create `deployment_handler_execution_time` histogram");
+        let eth_call_execution_time = registry
+            .new_deployment_histogram_vec(
+                "deployment_eth_call_execution_time",
+                "Measures the execution time for eth_call",
+                subgraph,
+                vec![String::from("method"), String::from("contract")],
```

Comment:
Maybe we should track the contract address also besides the name?

Reply:
Yes particularly for dynamic data sources I think tracking the address will be helpful

Reply:
done